packages/genesys-spark-components/src/components/stable/gux-icon/gux-icon.types.ts

```diff
@@ -1523,11 +1523,10 @@ export type GuxIconIconName =
   | 'refresh'
   | 'reset'
   | 'resource'
+  | 'restricted/custom/channels'
   | 'restricted/fa/building-user-regular'
   | 'restricted/fa/calendar-circle-user-regular'
   | 'restricted/fa/compass-drafting-regular'
-  | 'restricted/fa/grid-2-regular'
-  | 'restricted/fa/headset-regular'
```

Comment:
Should these have been removed ?

Reply:
Yes, it's part of the ticket and I confirmed with Daragh that these could be taken out.